Src/Plugins/Tests.Security/TestData/ReviewPotentiallySensitiveData/ExpectedOutputs/SEC102_002.SocialSecurityNumber_with_social_security_numbers.sarif

```diff
@@ -71,10 +71,10 @@
           ],
           "fingerprints": {
             "AssetFingerprint/v1": "",
-            "ValidationFingerprint/v1": "[secret=626B314E07BDE35B105543A8A75E2D257C9D4B2EB75B8BC7166489B6B01D19D3]",
-            "ValidationFingerprintHash/v1": "7550a76d0dcf856cd9325ff0f8dd10dfe306d35aa0f0973726381cbf0ae70a16"
+            "ValidationFingerprint/v1": "[secret=123-45-6789]",
```

Comment:
>123 [](http://example.com/codeflow?start=49&length=3)

another improvement, burn the secret into the fingerprint rather than the file hash!